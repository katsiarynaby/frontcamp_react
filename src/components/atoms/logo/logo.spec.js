import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';

import Logo from './index';

describe('Logo', () => {
    it("Logo should render correctly", () => {
        const wrapper = shallow(<Logo />);

        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});
