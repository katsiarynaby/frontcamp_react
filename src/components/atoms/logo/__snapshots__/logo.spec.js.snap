// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Logo should render correctly 1`] = `
<p
  className="logo"
>
  NETFLIX
  <span>
    roulette
  </span>
</p>
`;
